releasenotes/notes/trace-normalization-follow-up-c7a22fc4ae306f83.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    For trace normalization, use clearer default names `unnamed-service` and `unnamed-operation` instead of previously
```

Comment:
The change log only needs to reflect changes between releases, not between PRs. I think you just need to edit the old reno.

Reply:
Makes sense, updated